src/app/docs/id/id-docs.component.html

```diff
@@ -86,11 +80,7 @@
         headingStyle="success"
       >
         <p>
-          Use the ID directive when one HTML element must be associated with another HTML by its
-          <sky-code>id</sky-code> attribute, such as the <sky-code>label</sky-code> element's
-          <sky-code>for</sky-code> attribute. Hard-coding an ID on an element can cause ID
-          collisions if another element on the page has the same ID, leading to errors with
-          functionality and accessibility. Using the ID directive guarantees a unique ID that will not conflict with another element on the page.
+          Use the ID directive when one HTML element must be associated with another HTML element by its <sky-code>id</sky-code> attribute, such as the <sky-code>label</sky-code> element's <sky-code>for</sky-code> attribute. Use the ID directive to ensure unique IDs do not conflict with other elements on the page.
```

Comment:
For points 1, 3, and 4, I can make those changes. As for point 2:

> @Blackbaud-PaulCrowder, is id an HML attribute that the directive sets under the hood? Making id a code snippet makes it seems like that's the case, but in the first Don't use when item, we use ID uppercase and not as a code snippet. I don't see an id attribute in the code samples, so I'm not sure if this should actually be a code snippet.

Adding `skyId` to an element will generate an `id` attribute on the element at runtime. So this:

```
<input skyId>
```

Becomes something like this:

```
<input id="sky-id-gen__1">
```

In a simple HTML page, a developer would specify an `id` property, but this is abstracted away, so it's still necessary to talk about the `id` attribute because under simpler circumstances that's what the developer would be using.

Reply:
Makes sense. I figured something like that was going on but wanted to make sure. Thanks.